### PR TITLE
#246 added AE examples

### LINKED ISSUE #246: Expand `derive_extreme_flag()` examples
From the demo 2 user feedback: "baseline flag is OK but otherwise not enough documentation".

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

closes #246

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@bundfussr got it to work after and was able to test the site documentation update and looks good to me.

File: R/derive_var_extreme_flag.R [hunk 1]
```diff
@@ -459,6 +459,35 @@ derive_worst_flag <- function(dataset,
...
+#' # Most severe AE first occurrence per patient
+#' ae %>%
+#'   mutate(TEMP_AESEVN =
+#'          as.integer(factor(AESEV, levels = c("SEVERE","MODERATE","MILD")))) %>%
+#'   derive_var_extreme_flag(
+#'     new_var = AOCCIFL,
+#'     by_vars = vars(USUBJID),
+#'     order = vars(TEMP_AESEVN, AESTDY, AESEQ),
+#'     mode = "first"
+#'   ) %>%
```
Comment (bundfussr) [inline R/derive_var_extreme_flag.R:475]: I would add
```
select(-TEMP_AESEVN) %>%
```
to make the derivation complete.

File: R/derive_var_extreme_flag.R [hunk 2]
```diff
@@ -459,6 +459,35 @@ derive_worst_flag <- function(dataset,
...
+#' # Most severe AE first occurrence per patient per body system
+#' ae %>%
+#'   mutate(TEMP_AESEVN =
+#'          as.integer(factor(AESEV, levels = c("SEVERE","MODERATE","MILD")))) %>%
+#'   derive_var_extreme_flag(
+#'     new_var = AOCCSIFL,
+#'     by_vars = vars(USUBJID, AEBODSYS),
+#'     order = vars(TEMP_AESEVN, AESTDY, AESEQ),
+#'     mode = "first"
+#'   ) %>%
```
Comment (bundfussr) [inline R/derive_var_extreme_flag.R:488]: As above

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: R/derive_var_extreme_flag.R (hunk 1 above)
Comment (rossfarrugia) [reply R/derive_var_extreme_flag.R:475]: @bundfussr as this is just an example to show how the function works is that really needed? i would agree with that for in the ADaM templates but i think its overkill in this respect as the temp variable is not displayed anyway.

**Comment by bundfussr** [comment]
@thomas-neitmann , @bms63 , could you merge this PR using admin privileges?

**Comment by rossfarrugia** [comment]
@bundfussr i did this.

**Pull request merged**